controllers/userController.js

```diff
@@ -221,4 +217,22 @@ export const handleAdminDelete = async (req,res) => {
         message: "You are not admin",
     });
   }
+}
+
+export const handleAdminGetAllUser = async (req,res) => {
+  if(req.user?.role === "admin"){
+    const id = req.params.id;
+    const users = await User.find({role : "user"});
+    return res
```

Comment:
Handle Error using ErrorHandler if there is no users present, return "No users Found", You can get example at above Functions for how to handle errors